pack.go

```diff
@@ -369,6 +384,22 @@ func pushManifest(ctx context.Context, pusher content.Pusher, manifest any, medi
 	return manifestDesc, nil
 }
 
+// pushCustomConfig generates and pushes a custom config blob.
+func pushCustomConfig(ctx context.Context, pusher content.Pusher, mediaType string, annotations map[string]string) (ocispec.Descriptor, error) {
```

Comment:
It is not custom config but legacy empty config.

Reply:
maybe `customEmptyConfig` 😀